railties/lib/rails/console/app.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+require "rails/console/methods"
```

Comment:
Should we also add a deprecation warning here so there's a notice before removal in 8.0?

Reply:
Yes, let's add a deprecation to the individual files as well, otherwise someone importing it might not realize the files themselves are also going away.

Reply:
I actually committed it last week but somehow lost it when switching branches 😅